### PR TITLE
chore: add contributing

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

**Review by mariux — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- These points together mean that we may not have enough bandwidth to review and
+  integrate outside PRs right now. This will change in the future.
+
+You're welcome to send PRs! If we have time, or if the PRs are very small or fix
+bugs, we may even integrate them in the near future. But be aware that we might
+not get to it for a while, by which time it might no longer be relevant.
+
+If you want to ask about whether a PR is consistent with our short-term plan
+_before_ you put in the work -- and you should! -- Create an issue on the project
+so we can discuss it together.
```
[the remark this answers is not in the document]
Comment (mariux) [reply CONTRIBUTING.md:35]: that said... ofc we will focus on fixing bugs asap and also discuss and prioritize feature requests from the community once a week..

[the remark this answers is not in the document]
Comment (katcipis) [reply CONTRIBUTING.md]: simplified it

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Also we will not be able to review and integrate PRs for
+new features from the community, since those tend to be larger.
+This will change in time.
+
+If you want to ask about whether a PR is consistent with our short-term plan
+_before_ you put in the work -- and you should! -- Create an issue on the project
+so we can discuss it together.
+
+Feature requests and bug reports in form of issues are also welcomed.
```
Comment (soerenmartius) [inline CONTRIBUTING.md:17]:
```suggestion
First and foremost, we'd like to express our gratitude to you for taking the time to contribute. We welcome and appreciate any and all contributions via [Pull Requests](https://github.com/mineiros-io/terramate/pulls) along the [GitHub Flow](https://guides.github.com/introduction/flow/).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Thanks!
+
+## Contribution Workflow
+
+Terramate uses the “fork-and-pull” development model. Follow these steps if
+you want to merge your changes to Terramate:
```
Comment (soerenmartius) [inline CONTRIBUTING.md:24]:
```suggestion
For bug reports or requests, please submit your issue in the appropriate repository.

We advise that you open an issue and ask the [CODEOWNERS](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) and community prior to starting a contribution. This is your chance to ask questions and receive feedback before writing (potentially wrong) code. We value the direct contact with our community a lot, so don't hesitate to ask any questions.

For contributing to Terramate, please follow these steps:
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (katcipis) [reply CONTRIBUTING.md:17]: I thought we agreed that we would make it clear that we are not accepting PRs for new features for now, the new text doesn't make that clear, is that the way we want to go ? @mariux @soerenmartius

Comment (soerenmartius) [reply CONTRIBUTING.md:17]: sorry my bad - I just adjusted the next! Thank You!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+For bug reports or requests, please submit your issue in the appropriate repository.
+
+We advise that you open an issue and ask the [CODEOWNERS](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) and community prior to starting a contribution. This is your chance to ask questions and receive feedback before writing (potentially wrong) code. We value the direct contact with our community a lot, so don't hesitate to ask any questions.
+
+For contributing to Terramate, please follow these steps:
+
+1. Within your fork of
+   [Terramate](https://github.com/mineiros-io/terramate), create a
+   branch for your contribution. Use a meaningful name.
+1. Create your contribution, meeting all
```
Comment (mariux) [inline CONTRIBUTING.md:22]:
```suggestion
2. Create your contribution, meeting all
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+   [contribution quality standards](#contribution-quality-standards)
+1. [Create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
```
Comment (mariux) [inline CONTRIBUTING.md:24]:
```suggestion
3. [Create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+   against the main branch of the Terramate repository.
+1. Work with your reviewers to address any comments and obtain a
```
Comment (mariux) [inline CONTRIBUTING.md:26]:
```suggestion
4. Work with your reviewers to address any comments and obtain a
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+   minimum of 1 approval.
+1. Once the pull request is approved, one of the maintainers will merge it.
```
Comment (mariux) [inline CONTRIBUTING.md:28]:
```suggestion
5. Once the pull request is approved, one of the maintainers will merge it.
```

**Review by mariux — CHANGES_REQUESTED** [review]
let's keep it readable also in plain text ;)

File: CONTRIBUTING.md (hunk 4 above)
Comment (soerenmartius) [reply CONTRIBUTING.md:22]: that change is really not necessary - markdown is creating the continues list for us :) but ok for readability

Comment (soerenmartius) [reply CONTRIBUTING.md:22]: we should not block the PR because of something like this tho I assume

**Review by soerenmartius — APPROVED** [review]
lgtm!

**Review by i4ki — APPROVED** [review]
LGTM

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**